prac_08/box_layout.kv

```diff
@@ -0,0 +1,16 @@
+BoxLayout:
+    orientation: 'vertical'
+#    orientation: 'horizontal'
+    Button:
+        text: 'Clear'
+        on_press: app.handle_clear()
+    TextInput:
+        id: input_name
+        text: ''
+    Button:
+        text: 'Greet'
+        on_press: app.handle_greet()
```

Comment:
Could possibly pass through the text from input_name into the handle_greet function. Not a big thing, but could make the handle_greet function more generic.

Reply:
Yes, your absolutely right 👍